classes/Loggly.cls

```diff
@@ -462,32 +462,50 @@ global with sharing class Loggly {
 	*
 	* @param logs A set of log messages (in JSON format) to send to endpoint
 	*/
-	@future (callout=true)
 	global static void sendLogs(Set<String> logs) {
-		List<String> logList = new List<String>();
-		logList.addAll(logs);
+		try {
+			if (Limits.getQueueableJobs() < Limits.getLimitQueueableJobs()) {
+				System.enqueueJob(new QueueableLogSender(logs));
+			} else {
+				System.debug('Reached queueable limit');
+			}
+		} catch (Exception e) {
```

Comment:
I'm torn too on if it should be required to be caught by the calling class, but given that it's going to be handled the same way, I would just change this to be

```java
} catch (System.AsyncException e) {
    System.debug(System.LoggingLevel.ERROR, 'SLoggly: Unable to queue log message.\n Reason: ' + e.getMessage());
}
```